### PR TITLE
Offer factory choose random cover image from placeimg.com

### PR DESCRIPTION
**Pull request by magul**
__Description:__

Offer factory choose random cover image from placeimg.com
__New imports / dependencies:__

* `requests`
* `tqdm`

__Unit Tests:__

* N/A

__What tests do I need to run to validate this change:__

* Populate database using factories:
```
python3 manage.py populate_database
```
* Check, how new offer looks like in frontend.

### CODE REVIEW

**Comment by magul** [comment]
@juliaanholcer this pull reqest moves random images from frontend to content generation using factories - in such way user will alway get the same image, but different offers will have different, randomly choosen images.

@CorneeSean what do You think about it? It will also avoid mess in frontend that's resposible for choosing random images.

**Comment by CorneeSean** [comment]
I guess it is a better idea than putting it on front-end. Now when I think about it that was my first thought that it should be done in populate-database command, but I didn't feel confident enough to be able to code it by myself there :)

File: backend/apps/volontulo/factories.py [hunk 1]
```diff
@@ -132,6 +136,33 @@ class Meta:  # pylint: disable=C0111
...
 
+def placeimg_com_download(width, height, category):
+    """"placeimg.com downloader generator.
+
+    You can choose width, height and category and it will return argumentless
+    callable that will donwload such images and return file-like object.
+    """
+
+    def wrapped_func():
+        """Actural callable responsible for downloading images."""
```
Comment (CorneeSean) [inline backend/apps/volontulo/factories.py]: I guess you probably meant "Actual"?

Comment (magul) [reply backend/apps/volontulo/factories.py]: @CorneeSean fixed!

**Pull request merged**